### PR TITLE
Rename RESTWrapperOfSkipService to SkipServiceBroker and document it

### PR DESCRIPTION
**Pull request by jberdine**
- Rename RESTWrapperOfSkipService to SkipServiceBroker

This class provides methods that mirror and have thin implementations over
the skip service HTTP interface. This class is providing a method-call
interface for the HTTP interface. In this context, the term "broker" seems
more appropriate and clear.

- Document SkipServiceBroker

### CODE REVIEW

File: skipruntime-ts/helpers/src/rest.ts [hunk 1]
```diff
@@ -79,30 +104,55 @@ export class RESTWrapperOfSkipService {
...
+   * @returns {void}
+   */
   async put<K extends Json, V extends Json>(
     collection: string,
     key: K,
-    value: V[],
+    values: V[],
   ): Promise<void> {
-    return await this.patch(collection, [[key, value]]);
+    return await this.patch(collection, [[key, values]]);
```
Comment (jberdine) [inline skipruntime-ts/helpers/src/rest.ts:120]: The HTTP interface supports PUT, so I don't know why this is not implemented using it. Also, deleteKey could be implemented using PUT as well. Is there any reason to prefer the PATCH implementations? @beauby ?

Comment (beauby) [reply skipruntime-ts/helpers/src/rest.ts:120]: It might be to circumvent the key type issue. @skiplabsdaniel?

Comment (jberdine) [reply skipruntime-ts/helpers/src/rest.ts:120]: Ah, that looks related. I guess we should remove PUT when switching GET to POST.

**Comment by jberdine** [comment]
Does anyone have comments on renaming RESTWrapperOfSkipService to SkipServiceBroker?

**Comment by bennostein** [comment]
Sounds good to me!

**Review by bennostein — APPROVED** [review]

**Pull request merged**